moreinterp/go.mod

```diff
@@ -1,3 +1,17 @@
 module mvdan.cc/sh/moreinterp
 
-go 1.23
+go 1.24.0
+
+toolchain go1.24.4
+
+require (
+	github.com/u-root/u-root v0.14.1-0.20250722142936-bf4e78a90dfc
```

Comment:
I'm importing from this branch for now because it has some important fixed:

* https://github.com/u-root/u-root/pull/3421